doc/source/structures/vessels/crafttemplate.rst

```diff
@@ -0,0 +1,105 @@
+.. _crafttemplate:
+
+CraftTemplate
+=============
+
+.. structure:: CraftTemplate
+
+    You can access :struct:`CraftTemplate` objects from the :struct:`KUniverse`
+    bound variable.  Templates can be used to launch new vessels, or read initial
+    data about a craft, such as the description.
+
+    .. list-table::
+        :header-rows: 1
+        :widths: 1 1 2
+
+        * - Suffix
+          - Type
+          - Description
+
+        * - :attr:`NAME`
+          - :struct:`String`
+          - Name of this craft template
+        * - :attr:`FILEPATH`
+          - :struct:`String`
+          - The path to the craft file
+        * - :attr:`DESCRIPTION`
+          - :struct:`String`
+          - The description as saved in the editor
+        * - :attr:`EDITOR`
+          - :struct:`String`
+          - The editor where this craft was saved
+        * - :attr:`LAUNCHSITE`
+          - :struct:`String`
+          - The default launch site for this craft
+        * - :attr:`MASS`
+          - :struct:`Scalar`
+          - The default mass of the craft
+        * - :attr:`COST`
+          - :struct:`Scalar`
+          - The default cost of the craft
+        * - :attr:`PARTCOUNT`
+          - :struct:`Scalar`
+          - The total number of parts n this craft.
+
+
+.. attribute:: CraftTemplate:NAME
+
+    :access: Get only
+    :type: :struct:`String`
+
+    Returns the name of the craft.  It may differ from the file name.
```

Comment:
This brought me to some interesting reading.  I assume that you are referring to the two spaces between sentences.  That was how I was originally taught to type over 20 years ago.  In various articles on the topic, it now appears that the two space rule has fallen out of favor in some cases.  It was apparently originally tied to the use of monospaced fonts on typewriters, as extra space was needed to make sentence separation more recognizable with the extra white space in the words.  Today many style guides point to single spaces after sentences, though no one can point to firm reason or study why other than the majority of type setters thinking it looks better.  Of note however is the APA style guide which returned to a two space recommendation in their 6th edition.

But none of that is really the issue here.  Honestly, I don't know that I can break my habit of double spaces as I'm typing at this point.  I know I did it anywhere that I had multiple sentences, and I don't think it's worth trying to go back and correct every instance of it in the doc source.  The rendering engine will fix it for us, and we can't do a simple "replace all" on the source since spaces are used for indentation as well.

If you'd like to remove the extra spaces before you merge, that's fine by me.